.buildkite/pipelines/serverless.yml

```diff
@@ -105,7 +105,7 @@ steps:
       queue: n2-4-spot
     depends_on: build
     timeout_in_minutes: 40
-    parallelism: 16
+    parallelism: 12
```

Comment:
For pull requests: https://github.com/elastic/kibana/blob/d058dd59d8ebb168ed15b0b045397eb615eb257b/.buildkite/pipelines/pull_request/base.yml#L137-L148

Reply:
Thanks! 👍